packages/skill-lesson/video/video-overlays.tsx

```diff
@@ -652,12 +653,14 @@ const BuyProduct: React.FC<{product?: SanityProduct}> = ({product}) => {
                         href={`/bonuses/${module.slug}`}
                         target="_blank"
                       >
-                        <Image
-                          src={module.image?.url ?? module.image}
-                          alt={`${module.title} workshop`}
-                          width={60}
-                          height={60}
-                        />
+                        {(module.image?.url || module.image) && (
+                          <Image
+                            src={module.image?.url ?? module.image}
+                            alt={`${module.title} workshop`}
+                            width={60}
+                            height={60}
+                          />
+                        )}
```

Comment:
I likely didn't have set up working correctly so data wasn't there but added a check for image url as a fail-safe. Was running into rendering errors getting started